gamedata/alldefs_post.lua

```diff
@@ -816,8 +919,24 @@ function UnitDef_Post(name, uDef)
 					scavboss = 1000,
 					commanders = 0,
 					}
+			uDef.weapondefs = {
+				repulsor = {
+					name = "DRepulsor",
+					range = 50,
+					weapontype = "Shield",
+					shield = {
+						interceptType = 32,
+						smart = true,
+						radius = 50,
+						visibleHitFrames = 60
+						},
+				},
+			}
+			uDef.weapons[4] = {
+				def = "REPULSOR",
+				}
 			end
```

Comment:
Is this section supposed to be commented out?

Reply:
Yes, it was from when I tried to make it work as a shield instead of removing the projectile. Actually, I'll remove it before merging.